spec/helpers/sufia_helper_spec.rb

```diff
@@ -1,3 +1,7 @@
+def new_state
```

Comment:
I see. I grepped for `new_state` and didn't see it in the code before, but now I see you reworked it. OK, I'm cool with this.